configure.py

```diff
@@ -531,13 +534,13 @@ def has_re2c():
 if platform.is_aix():
     objs += cc('getopt')
 if platform.is_msvc():
-    ninja_lib = n.build(built('ninja.lib'), 'ar', objs)
+    ninja_lib = n.build(built('ninjalib.lib'), 'ar', objs)
```

Comment:
What's prompting this?

Reply:
Reverted. This was from simdjson, which has some dllexports. These caused the configure.py bootstrap build to create a ninja.exp and ninja.lib next to the ninja.exe. That's fine for the first time you build, but on subsequent builds the link step would fail because nina.lib would no longer resolve to the ninja.lib in the build directory.